.circleci/config.yml

```diff
@@ -215,10 +215,6 @@ workflows:
           dynamic_hostname: true
           requires:
             - build
-          filters:
-            branches:
-              ignore:
-                - master
```

Comment:
We don't want multideploy uat deploys from master though, right?

Reply:
Good point, I guess master can skip multi-deploy uat.
Do you think it should be deployable to static-uat?

Reply:
Yep